elements/lisk-dpos/src/delegates_list.ts

```diff
@@ -51,10 +47,23 @@ interface DelegatesListConstructor {
 }
 
 interface DelegateListWithRoundHash {
-	readonly address: string;
+	readonly address: Buffer;
 	roundHash: Buffer;
 }
 
+export const decodeForgerList = (buffer: Buffer): ForgersList => {
+	// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+	const parsedForgersList = JSON.parse(buffer.toString('utf8'));
+	// eslint-disable-next-line
+	return parsedForgersList.map(
+		(fl: { round: number; delegates?: string[]; standby?: string[] }) => ({
+			round: fl.round,
+			delegates: fl.delegates?.map(d => Buffer.from(d, 'binary')) ?? [],
```

Comment:
As discussed everywhere we present `Buffer -> String` we will use Base64. Some should use that consistent approach everywhere...

Reply:
The Buffer usage will be update with https://github.com/LiskHQ/lisk-sdk/issues/5266 issue